akka-docs/src/main/paradox/general/stream/stream-design.md

```diff
@@ -90,11 +90,11 @@ Exceptions from this need to be well-justified and carefully documented.
 
 Akka Streams must enable a library to express any stream processing utility in terms of immutable blueprints. The most common building blocks are
 
- * Source: something with exactly one output stream
```

Comment:
`handleWith` mentioned above. There are many overloaded methods `handleWith`. Which should I take? Is it important to add link?

Reply:
No, let's skip that.